qwiic_buzzer.py

```diff
@@ -0,0 +1,565 @@
+#-------------------------------------------------------------------------------
+# Written by SparkFun Electronics, January 2024
+#
+# This python library supports the SparkFun Electroncis Qwiic ecosystem
+#
+# More information on Qwiic is at https://www.sparkfun.com/qwiic
+#
+# Do you like this library? Help support SparkFun. Buy a board!
+#===============================================================================
+# Copyright (c) 2024 SparkFun Electronics
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#===============================================================================
+
+"""
+qwiic_buzzer
+============
+Python module for the [SparkFun Qwiic Buzzer](https://www.sparkfun.com/products/24474)
+This is a port of the existing [Arduino Library](https://github.com/sparkfun/SparkFun_Qwiic_Buzzer_Arduino_Library)
+This package can be used with the overall [SparkFun Qwiic Python Package](https://github.com/sparkfun/Qwiic_Py)
+New to Qwiic? Take a look at the entire [SparkFun Qwiic ecosystem](https://www.sparkfun.com/qwiic).
+"""
+
+
+# The Qwiic_I2C_Py platform driver is designed to work on almost any Python
+# platform, check it out here: https://github.com/sparkfun/Qwiic_I2C_Py
+import qwiic_i2c
+import time
+
+# Define the device name and I2C addresses. These are set in the class defintion
+# as class variables, making them avilable without having to create a class
+# instance. This allows higher level logic to rapidly create a index of Qwiic
+# devices at runtine
+_DEFAULT_NAME = "Qwiic Buzzer"
+
+# Some devices have multiple available addresses - this is a list of these
+# addresses. NOTE: The first address in this list is considered the default I2C
+# address for the device.
+_DEFAULT_ADDRESS = 0x34
+_FULL_ADDRESS_LIST = list(range(0x08, 0x77+1))  # Full address list (excluding reserved addresses)
+_FULL_ADDRESS_LIST.remove(_DEFAULT_ADDRESS >> 1)   # Remove default address from list
```

Comment:
Don't do the bit shifting here, because that removes 0x1A instead of 0x34.

Reply:
oops. removed